package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "next": "12.2.5",
     "next-seo": "^5.5.0",
+    "pnpm": "^7.9.5",
```

Comment:
Remove this package as we install it globally in our OS

Reply:
Removed. Thanks for the comment.